workflow_array_ephys/ingest.py

```diff
@@ -9,38 +9,44 @@
 import element_data_loader.utils
 
 def ingest_subjects(subject_csv_path='./user_data/subjects.csv'):
+    """
+    Ingest subjects listed in the subject column of of ./user_data/subjects.csv
+    """
     # -------------- Insert new "Subject" --------------
     with open(subject_csv_path, newline= '') as f:
         input_subjects = list(csv.DictReader(f, delimiter=','))
-
+    # Broz 102821 - this gives full # even if skipped, not # populated
     print(f'\n---- Insert {len(input_subjects)} entry(s) into subject.Subject ----')
     subject.Subject.insert(input_subjects, skip_duplicates=True)
 
+    print('\n---- Successfully completed ingest_subjects ----')
 
 def ingest_sessions(session_csv_path='./user_data/sessions.csv'):
-    root_data_dir = get_ephys_root_data_dir()
-
+    """
+    Ingests SpikeGLX and OpenEphys files from directories listed
+    in the sess_dir column of ./user_data/sessions.csv
+    """
     # ---------- Insert new "Session" and "ProbeInsertion" ---------
     with open(session_csv_path, newline= '') as f:
         input_sessions = list(csv.DictReader(f, delimiter=','))
 
     # Folder structure: root / subject / session / probe / .ap.meta
-    session_list, session_dir_list, probe_list, probe_insertion_list = [], [], [], []
+    session_list, sess_dir_list, probe_list, probe_insertion_list = [], [], [], []
 
     for sess in input_sessions:
-        session_dir = element_data_loader.utils.find_full_path(
-                                                    get_ephys_root_data_dir(), 
+        sess_dir = element_data_loader.utils.find_full_path(
```

Comment:
```suggestion
        session_dir = element_data_loader.utils.find_full_path(
```

I would suggest keeping the variable name `session_dir`.

Reply:
Sure - I'll make sure this is consistent across the workflow